lms/static/sass/shared/_footer-edx.scss

```diff
@@ -321,6 +430,100 @@ footer#footer-edx-v3 {
   @include media-breakpoint-up(sm) {
     padding: $baseline $baseline*0.5;
 
+    .column-1 {
+      flex: 0 0 58.33333%;
+      max-width: 58.33333%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2 {
+      flex: 0 0 50%;
+      max-width: 50%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-3 {
+      flex: 0 0 50%;
+      max-width: 50%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-2-1 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+      //flex-basis: auto;
+    }
+
+    .column-2-2 {
+      flex: 0 0 80%;
+      max-width: 80%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+
+      @include margin-left(10%);
+      //flex-basis: auto;
+    }
+
+    .column-3-1 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
+
+    .column-3-2 {
+      flex: 0 0 100%;
+      max-width: 100%;
+      position: relative;
+      width: 100%;
+      min-height: 1px;
+      padding-right: 15px;
+      padding-left: 15px;
+    }
```

Comment:
These are all shared between the classes;
```
position: relative;
width: 100%;
min-height: 1px;
padding-right: 15px;
padding-left: 15px;
```
Please update to be more DRY (e.g. with a common class)

Reply:
This is the same as the default settings, it shouldn't need to be updated?

Reply:
yes it will not change for any column